docs/modules/release-notes/pages/5-3-0.adoc

```diff
@@ -0,0 +1,231 @@
+You can set these options using either the advanced network configurations or Hazelcast system properties.
+See xref:clusters:network-configuration.adoc#configuring-tcp-keep-alive[Configuring TCP Keep-Alive].
+https://github.com/hazelcast/hazelcast/pull/23816[#23816], https://github.com/hazelcast/hazelcast/pull/24050[#24050]
+
+=== SQL Engine
+
+* The `CREATE MAPPING` statement for Compact Serialization now supports nullable primitives.
+https://github.com/hazelcast/hazelcast/pull/24300[#24300]
+* Implemented the `GET_DDL` function.
+https://github.com/hazelcast/hazelcast/pull/23595[#23595]
+* Implemented the `SHOW DATA CONNECTIONS` statement to show the connections created by the `CREATE CONNECTIONS` statement.
+https://github.com/hazelcast/hazelcast/pull/24028[#24028]
+* Added support of providing declarative partitioning strategies based on attributes. These attributes represent specific partition keys for a map.
+You can also configure this strategy dynamically.
+https://github.com/hazelcast/hazelcast/pull/23535[#23535], https://github.com/hazelcast/hazelcast/pull/23954[#23954]
+* Improved the nested loop joins algorithm by making the `CreateDagVisitor` interface generic.
+https://github.com/hazelcast/hazelcast/pull/23368[#23368]
+* Implemented the `TO_CHAR` function which is capable of formatting dates, timestamps, and numbers.
+https://github.com/hazelcast/hazelcast/pull/22529[#22529]
+* Implemented `JSON=VARCHAR` comparison.
+https://github.com/hazelcast/hazelcast/pull/22746[#22746]
+
+=== Jet Engine
+
+* Each Jet engine job was creating a client for connecting to the remote cluster, resulting in an excessive number of clients.
+This has been fixed by introducing a single data connection which can be reused for all job instances.
+https://github.com/hazelcast/hazelcast/pull/23472[#23472], https://github.com/hazelcast/hazelcast/pull/23849[#23849]
+* Added MongoDB read (source) and write (sink) processors.
+https://github.com/hazelcast/hazelcast/pull/23348[#23348]
+* Introduced `JobStatusListener` as an alternative to retrieve a job status via the `Job.getStatus()` method.
+https://github.com/hazelcast/hazelcast/pull/23193[#23193]
+* Updated the https://docs.hazelcast.org/docs/{page-latest-supported-java-client}/javadoc/com/hazelcast/jet/Job.html#isUserCancelled--[job API] to add the ability
+to distinguish the user-cancelled jobs from the failed ones.
+https://github.com/hazelcast/hazelcast/pull/22924[#22924]
+* Added `flock` to guard all the concurrent `pip` executions (upgrading `pip` and `protobuf` versions) in the Jet-to-Python script.
+https://github.com/hazelcast/hazelcast/pull/22829[#22829]
+
+=== Security
+
+* Added the TLS configuration property `keyMaterialDuration` which allows using a configurable timeout for caching keyStores and trustStores.
+See xref:security:tls-configuration.adoc[TLS Configuration] for the details on how to configure this property.
+https://github.com/hazelcast/hazelcast/pull/23307[#23307]
```

Comment:
as part of "security" (by obscurity) it is impossible now to create SQL mappings for `__sql.catalog` and `__jet.*` - technically a breaking change